climateconnect/climateconnect
# Feat/adapt share project third step

This PR finished up the necessary changes that need to be made to the "Share project" code to enable the event calendar (see #1189). 
It includes the following changes:
- Enter project name in step 3 of share project
- Use new Date pickers from upgraded Material ui - show different date options dependent on type. Added manual date validation because options like `required` aren't supported by the new package
- Allow putting exact locations (e.g. townhall San Francisco). Before it was only possible to specify the city
- Allow putting additional info to locations (e.g. which room in a building an event is in)

## To test:
- Run migrations
- Run `create_project_type_test_data` section of the `create_test_data` management script
- Go through the "Share project" flow once for each project type, see the different options
- Share one project of each type

## Icons used for the three project types to use in your local db
Project type icon
![project_type_icon](https://github.com/climateconnect/climateconnect/assets/10531079/6f1b16fe-19c1-42ed-9838-cab7857721e8)

Idea type icon
![idea_type_icon](https://github.com/climateconnect/climateconnect/assets/10531079/0c299fb1-45cf-4bb0-b722-2fed45ad88b3)

Event type icon
![event_type_icon](https://github.com/climateconnect/climateconnect/assets/10531079/f750958c-4d73-4da1-b53b-a4c5f88b2e82)
